### DISCUSSION TITLE
Notes - 06th Feb 2024

### DISCUSSION THREAD

**Discussion by mmcky** [QuantEcon/reading-group-2024#7] [General]
## Review of GDP Correlations

- Update plot for GDP per capita and Share of Household Consumption (Investment) to be based on %'s and look at correlation (Margaret)

### HDI

- Update plots to use `plotly`
- @mmcky to find HDI chapter in Economic Development textbook (Thirlwall?) and share with the group
- HDI is the geometric mean of three different dimension index.  

The income index is constructed as:

$$
I = \frac{ln(value) - ln(min)}{ln(max) - ln(min)}
$$

Here are the [technical notes](https://hdr.undp.org/system/files/documents/technical-notes-calculating-human-development-indices.pdf)

`jupyterlab-myst`

- @mmcky to setup a tutorial for new RA's and explain `myst`

**Comment by pgrosser1** [comment]
I will be working on the following changes:
- Adding secondary plots in the HDI section to show that there is a high correlation in all four factors of the HDI calculation (inc. mean & expected years of schooling)
- Including plots that can be hovered over to see the country name in all correlations
- Residual plot for energy. See if removal of last few data points changes quality of the fit
- Life expectancy at birth - break up the correlation for low vs. high income countries (expected to go from linear to no correlation at all)
- Add correlations with infant mortality
- Economic structure for Gini coefficient on plots could provide more insight

**Comment by HumphreyYang** [comment]
Many thanks @QuantEcon/reading-group-2024 for the great discussion.

Here are some notes I put into the chat during the meeting:

**1. Data Source for Macro Data**

Here is a nice source of data where you can find all FED-curated macro data
https://fred.stlouisfed.org/

A nice package to use in Python is
https://pandas-datareader.readthedocs.io/en/latest/remote_data.html

where it has direct access to FRED without the need for an API key.

We used this library in https://intro.quantecon.org/business_cycle.html#unemployment if you are looking for a quickstart example.

**2. Decomposition of GDP growth across countries**

If you are interested, we can find decompositions of GDP growth in 
https://databank.worldbank.org/embed/gdp-composition/id/fced70c?review=y

**3. Environmental indicators**

Here are some environmental indicators that might be helpful
https://www.oecd.org/env/indicators-modelling-outlooks/data-and-indicators.htm

**Comment by mbek0605** [comment]
For next week,  I will make the changes listed below: 

1. "Update plot for GDP per capita and Share of Household Consumption (Investment) to be based on %'s and look at correlation"
2. Examine % of the consumption of GDP for different countries 
3. Same for R&D as part of GDP 
4. Specify exchange rate unit 
5. Ensure that the title of graphs makes sense 
6. Make input into functions user-friendly 
7. Time plot with GDP per capita vs. average world temperature (??)

**Reply by jstac** [reply]
Thanks @mbek0605 .   I'm thinking of something like https://transportgeography.org/contents/chapter4/transportation-and-environment/average-global-temperature-world-carbon-emissions-fossil-fuel/ but with world GDP.    World GDP can be found here: https://intro.quantecon.org/long_run_growth.html#regional-analysis  (and we're cutting that graph from that section, so it's good if you can use it).
